core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala

```diff
@@ -144,7 +144,18 @@ abstract class AbstractEventsByTagSpec(override val systemName: String, config:
       val tagId = writePluginConfig.tags.getOrElse(tag, 1)
       bs.setString("tag" + tagId, tag)
     }
-    bs.setBytes("message", serialized)
+    val serializer = serialization.findSerializerFor(persistent)
+    val serManifest = serializer match {
+      case ser2: SerializerWithStringManifest ⇒
+        ser2.manifest(persistent)
+      case _ ⇒
+        if (serializer.includeManifest) persistent.getClass.getName
+        else PersistentRepr.Undefined
+    }
+    bs.setInt("ser_id", serializer.identifier)
+    bs.setString("ser_manifest", serManifest)
+    bs.setString("event_manifest", persistent.manifest)
+    bs.setBytes("event", serialized)
```

Comment:
ah, now I see what you did here. This is confusing, because this is not how they are stored by CassandraJournal. Here you store the `PersistentRepr` as the "event` but it is supposed to be `PersistentRepr.payload`, i.e. the actual event. See `serializeEvent` in `CassandraJournal`.